src/effects/native/reverbeffect.cpp

```diff
@@ -31,6 +31,7 @@ EffectManifest ReverbEffect::getManifest() {
     time->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
     time->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
     time->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
+    time->setDefaultLinkType(EffectManifestParameter::LINK_LINKED);
```

Comment:
I played around with it a bit more, and yeah, you're right.

IMO we could use some better sounding reverb effects, if anyone is up to the challenge.